src/components/Pagination/__tests__/index.test.js

```diff
@@ -0,0 +1,379 @@
+import { act, waitFor } from '@testing-library/react'
+import userEvent from '@testing-library/user-event'
+import React from 'react'
+import Pagination, { usePagination } from '..'
+import shouldMatchEmotionSnapshot from '../../../helpers/shouldMatchEmotionSnapshot'
+import getPageNumbers from '../getPageNumbers'
+
+const ExampleChildren = () => {
+  const { pageData } = usePagination()
+
+  return (
+    <ul>
+      {pageData.map(item => (
+        <li key={item}>{item}</li>
```

Comment:
isn't `item` an array ?

Reply:
No, because `pageData` is the array of the current page @matthprost